customcommands/bot.go

```diff
@@ -156,7 +156,9 @@ var cmdEvalCommand = &commands.YAGCommand{
 		ctx := templates.NewContext(data.GuildData.GS, channel, data.GuildData.MS)
 		ctx.IsExecedByEvalCC = true
 
-		code := common.ParseCodeblock(data.Args[0].Str())
+		// Access stripped message directly, so Dcmd argument parsing can't mess with the user's code
+		// code := common.ParseCodeblock(data.Args[0].Str())
```

Comment:
I would suggest something along the lines of
```suggestion
		// use stripped message content instead of parsed arg data to work
		// around dcmd bug where input backslashes, spaces are lost; see
		// https://github.com/botlabs-gg/yagpdb/pull/1547

		// TODO: fix dcmd and switch back to using `data.Args[0].Str()`
```
This avoids the nebulous "mess with the user's code" while still remaining acceptably brief IMO.

Reply:
Hm. Upon further reflection I think framing the issue at hand as a dcmd bug isn't entirely accurate; while losing spaces is unexpected, the escaping behavior with backslashes is intended. So even if we fix the issue with spaces, evalcc won't be able to use dcmd argument data, which is unfortunate.

Perhaps something like
```suggestion
		// use stripped message content instead of parsed arg data to avoid dcmd
		// from misinterpreting backslashes and losing spaces in input; see
		// https://github.com/botlabs-gg/yagpdb/pull/1547
```
would be more accurate in that case.

Reply:
kewl, thank you